pkg/cmd/codespace/ports.go

```diff
@@ -144,15 +144,15 @@ func getDevContainer(ctx context.Context, apiClient apiClient, codespace *api.Co
 	return ch
 }
 
-func newPortsPrivacyCmd(app *App) *cobra.Command {
+func newPortsVisibilityCmd(app *App) *cobra.Command {
 	return &cobra.Command{
-		Use:     "privacy <port:public|private|org>...",
-		Short:   "Change the privacy of the forwarded port",
-		Example: "gh codespace ports privacy 80:org 3000:private 8000:public",
+		Use:     "visibility <port:public|private|org>...",
```

Comment:
Let's improve the Use line according to our [cli syntax guidelines](https://github.com/cli/cli/blob/trunk/docs/command-line-syntax.md):
1. The `:` is a literal character, not part of the placeholder, therefore the placeholder should not include it: e.g. `<port>:`
2. Values `public|private|org` are literal, not placeholders; therefore they should be denoted with curly braces: `{public|private|org}`

So all together, I think the Use line should be:
```suggestion
		Use:     "visibility <port>:{public|private|org}...",
```
If you think this could be difficult to parse for a user, we could also consider `<port>:<visibility>` and use the docs to list valid values for "visibility".

Reply:
That looks great. We can change the docs further as we hear feedback. Thanks!